tests/x509/test_x509.py

```diff
@@ -4740,6 +4765,70 @@ def test_rfc4514_string_empty_values(self):
             ]
         )
         assert n.rfc4514_string() == "CN=cryptography.io,O=PyCA,L=,ST=,C=US"
+        assert n == x509.Name.from_rfc4514_string(n.rfc4514_string())
+
+    def test_rfc4514_parse_escapes(self):
+        name = r"C=US,CN=Joe \, Smith,DC=example"
+        n = x509.Name.from_rfc4514_string(name)
+        assert n == x509.Name(
+            [
+                x509.NameAttribute(NameOID.DOMAIN_COMPONENT, "example"),
+                x509.NameAttribute(NameOID.COMMON_NAME, "Joe , Smith"),
+                x509.NameAttribute(NameOID.COUNTRY_NAME, "US"),
+            ]
+        )
+        assert n.rfc4514_string() == name
+
+        # Quote characters also work
+        name = r"C=US,CN=Jane \"J\,S\" Smith,DC=example"
+        n = x509.Name.from_rfc4514_string(name)
+        assert n == x509.Name(
+            [
+                x509.NameAttribute(NameOID.DOMAIN_COMPONENT, "example"),
+                x509.NameAttribute(NameOID.COMMON_NAME, 'Jane "J,S" Smith'),
+                x509.NameAttribute(NameOID.COUNTRY_NAME, "US"),
+            ]
+        )
+        assert n.rfc4514_string() == name
+
+        # Not strictly part of the spec, but simple quotes work too:
+        name = 'C=US,CN="Jane J,S Smith",DC=example'
```

Comment:
Test also for `=` inside quotes...

Reply:
Done, and also for the case with no `=`, no surprises here.